website/docs/04-emulators/05-miscellaneous/15-superbroswar.md

```diff
@@ -18,9 +18,9 @@ slug: /emulators/superbroswar
 
 Super Mario War / Super Cat Wars is a fan-made multiplayer Super Mario Bros. Very customizable with many mods.
 Onion includes the Retroarch core. The standalone version (which includes the multiplayer mode) will be distributed as a port in the Onion Ports repository.
-The RetroArch core version allows to fight against enemies controled by the computer.
+The RetroArch core version allows to fight against enemies controlled by the computer.
```

Comment:
I know you didn't change this, but can this sentence read: 

`The RetroArch core version allows you to fight against enemies controlled by the computer.`

Reply:
Yes, better so